flask_whooshee.py

```diff
@@ -11,8 +11,9 @@
 import whoosh.index
 import whoosh.qparser
 
-from flask_sqlalchemy import models_committed, BaseQuery
-from sqlalchemy import text
+from flask import current_app
```

Comment:
Is this import really necessary? I don't see it used anywhere else in the patch.